src/pages/components/CertificateInputFields.js

```diff
@@ -0,0 +1,185 @@
+          <FlexLargeDiv>
+            <ContainerFlexNoWrapDiv>
+              <GrowingDiv>
+                <InputHeadingDiv>Start timestamp of energy use</InputHeadingDiv>
+                <InputWrapLeftDiv>
+                  <InputHalfWrap>
+                    <InputFieldDate
+                      val={sdVal}
+                      onChangeVal={handleSdChgeVal}
+                      name="sd"
+                      id="sd"
+                    />
+                  </InputHalfWrap>
+                  <InputHalfWrap>
+                    <InputFieldTime
+                      val={stVal}
+                      onChangeVal={handleStChgeVal}
+                      name="st"
+                      id="st"
+                    />
+                  </InputHalfWrap>
+                </InputWrapLeftDiv>
+              </GrowingDiv>
+              <CertificateTimeInterval
+                sTimestamp={`${sdVal} ${stVal}`}
+                eTimestamp={`${edVal} ${etVal}`}
+              />
+              <GrowingDiv>
+                <InputHeadingDiv>End timestamp of energy use</InputHeadingDiv>
+                <InputWrapRightDiv>
+                  <InputHalfWrap>
+                    <InputFieldDate
+                      val={edVal}
+                      onChangeVal={handleEdChgeVal}
+                      name="ed"
+                      id="ed"
+                    />
+                  </InputHalfWrap>
+                  <InputHalfWrap>
+                    <InputFieldTime
+                      val={etVal}
+                      onChangeVal={handleEtChgeVal}
+                      name="et"
+                      id="et"
+                    />
+                  </InputHalfWrap>
+                </InputWrapRightDiv>
+              </GrowingDiv>
+            </ContainerFlexNoWrapDiv>
+          </FlexLargeDiv>
+          <FlexDiv>
+            <ContainerFullWidthWrapDiv>
+              <InputHeadingDiv>Electric energy use</InputHeadingDiv>
+              <InputWrapDiv>
+                <InputWrap>
+                  <InputFieldEnergy
+                    val={enVal}
+                    onChangeVal={handleEnChgeVal}
+                    name="en"
+                    id="en"
+                  />
+                </InputWrap>
+              </InputWrapDiv>
+            </ContainerFullWidthWrapDiv>
+          </FlexDiv>
+          <FlexDiv>
+            <ContainerFullWidthWrapDiv>
+              <InputHeadingDiv>H2 batch size</InputHeadingDiv>
+              <InputWrapDiv>
+                <InputWrap>
+                  <InputFieldSize
+                    val={szVal}
+                    onChangeVal={handleSzChgeVal}
+                    name="sz"
+                    id="sz"
+                  />
+                </InputWrap>
+              </InputWrapDiv>
+            </ContainerFullWidthWrapDiv>
+          </FlexDiv>
+        </ContainerFlexWrapDiv>
+      </MarginedDiv>
+    </>
+  )
+}
+
+const MarginedDiv = styled.div`
+  margin: 8px;
```

Comment:
generally speaking if you can use either margin or padding choose padding to avoid collapsing margins. Don't need to change but just a rule of thumb. COuld also use inset here as well I think

Reply:
Done.